src/templates/base/2019/contributors.html

```diff
@@ -354,15 +362,31 @@ <h1 class="title title-lg">
 {% block scripts %}
   {{ super() }}
   <script nonce="{{ csp_nonce() }}">
+  
+  // Randomise the contributors on reload as server-side random will be cached at CDN.
+  // Only run this if the page is reloaded for performance reasons.
+  
+  // Support both newer PerformanceNavigationTiming API for newer browsers
+  // and older Navigation Timing API for older browsers
+  if ((performance.getEntriesByType("navigation")[0] && performance.getEntriesByType("navigation")[0].type === "reload")
+      ||
+        (performance.navigation && performance.navigation.type === performance.navigation.TYPE_RELOAD)) {
+    var ul = document.querySelector('.contributor-grid');
+    for (var i = ul.children.length; i >= 1; i--) {
+        // Ensure position 1 is minimum to leave "Join the team" icon at beginning
+        ul.appendChild(ul.children[Math.floor(Math.random() * i) + 1]);
+    }
+  }
+
 (function loaded() {
```

Comment:
Nit: could you remove the parentheses from around the function now that we're not immediately invoking it?

Reply:
Done!